packages/reports/addon/templates/components/visualization-toggle.hbs

```diff
@@ -0,0 +1,12 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<span class='visualization-toggle'>
```

Comment:
We can remove this now.

Reply:
I remembered this as I was getting in my car to leave. whoops